oneview_redfish_toolkit/tests/blueprints/test_chassis.py

```diff
@@ -159,6 +159,34 @@ def test_get_enclosure_chassis(
         self.assertEqual(self.enclosure_chassis_mockup, json_str)
 
     @mock.patch.object(util, 'get_oneview_client')
+    def test_get_enclosure_chassis_with_etag(
+            self, get_oneview_client_mockup):
+        """"Tests EnclosureChassis and ETag response header"""
+
+        oneview_client = get_oneview_client_mockup()
+
+        oneview_client.index_resources.get_all.return_value = \
+            [{"category": "enclosures"}]
+        oneview_client.enclosures.get.return_value = self.enclosure
+        oneview_client.enclosures.get_environmental_configuration.\
+            return_value = self.enclosure_environment_configuration_mockup
+
+        # Get EnclosureChassis
+        response = self.app.get(
+            "/redfish/v1/Chassis/0000000000A66101"
+        )
+
+        json_str = response.data.decode("utf-8")
+
+        # Tests response
+        self.assertEqual(status.HTTP_200_OK, response.status_code)
+        self.assertEqual("application/json", response.mimetype)
+        self.assertEqual(self.enclosure_chassis_mockup, json_str)
+        self.assertEqual(
```

Comment:
I would just add this assert line to the enclosure_chassis test. This work, is OK but the other is optimized in my opinion. Same for the other tests.

Reply:
I kind of agree with this... as there should not be a enclosure chassis without an etag from now on.
Can you update that, @adriane-cardozo?